layouts/_default/baseof.html

```diff
@@ -34,6 +34,10 @@
       {{ end }}
       {{ $t.Stop }}
     {{ end }}
+    {{ $noop := .WordCount }}
+    {{ if .Page.Store.Get "hasMath" }}
+      <link href="https://cdn.jsdelivr.net/npm/katex@0.16.21/dist/katex.min.css" rel="stylesheet">
```

Comment:
Why do we need this?

Reply:
Never mind, I now see the Hugo extension issue.

Reply:
With the `transform.ToMath` output option set to `mathml` (the default) , with or without the CSS:

![image](https://github.com/user-attachments/assets/70af1f68-349b-4c81-9dc7-6c53f717ddff)

With the `transform.ToMath` output option set to `htmlAndMathml`, without the CSS:

![image](https://github.com/user-attachments/assets/772f3f98-2542-4cbc-a3e5-b333071db02a)

With the `transform.ToMath` output option set to `htmlAndMathml`, with the CSS:

![image](https://github.com/user-attachments/assets/626c9dbb-bbce-4817-97e4-97b268c20f2d)

Clearly the last is the best quality.

Reply:
OK, now I see; first time I read it, I thought you imported some JS file.